usl_pipeline/usl_lib/usl_lib/writers/polygon_writers.py

```diff
@@ -0,0 +1,40 @@
+import typing
+from typing import Tuple
+
+from shapely import geometry
+
+
+def num_list_to_strings(num_list):
+    return list(map(lambda v: str(v), num_list))
```

Comment:
the `" ".join` calls below will happily take the generator returned by `map`, so we can drop the `list` call here to avoid materializing a list.
Can also drop the lambda so this could just be
```python
return map(str, num_list)
```
Python folks also like to write this as comprehensions, so below we could also do
```python
" ".join(str(v) for v in num_list)
```

Reply:
Oh, you're right, I don't actually need to switch to list and to have the "num_list_to_strings" function at all! Removed it completely.